iOS Example Frame/iOS Example Frame/Style/CustomStyle1.swift

```diff
@@ -51,7 +51,7 @@ struct PayButtonFormStyleCustom1: ElementButtonStyle {
 // MARK: - Main Payment Form
 
 struct PaymentFormStyleCustom1: PaymentFormStyle {
-  var backgroundColor: UIColor = Constants.backgroundColor
+  var backgroundColor: UIColor = .white
```

Comment:
I am unsure if this is supposed to get committed? Makes sense you tested it, but CustomStyle1 is still matching a design

Reply:
There's no custom style 1 design. other is a duplicate for Matrix style. It just a test ground.